src/overlays/actors/ovl_En_Egblock/z_en_egblock.h

```diff
@@ -2,16 +2,44 @@
 #define Z_EN_EGBLOCK_H
 
 #include "global.h"
+// #include "overlays/actors/ovl_En_Egol/z_en_egol.h"
 
 struct EnEgblock;
 
 typedef void (*EnEgblockActionFunc)(struct EnEgblock*, PlayState*);
 
+#define GET_EGBLOCK_PARAM_7F(thisx) ((thisx)->params & 0x7F)
+#define GET_EGBLOCK_PARAM_F80(thisx) (((thisx)->params >> 7) & 0x1F)
+#define GET_EGBLOCK_TYPE(thisx) (((thisx)->params >> 12) & 0xF)
+
+#define EGBLOCK_PARAM(type, param1, param2) ((((type) & 0xF) << 12) | (((param1) & 0x1F) << 7) | ((param2) & 0x7F))
+
+#define EGBLOCK_TYPE_ACTIVE 0
+
+typedef struct {
+    /* 0x00 */ u8 active;
+    /* 0x04 */ Vec3f pos;
+    /* 0x10 */ s16 alpha;
+    /* 0x12 */ s16 timer;
+    /* 0x14 */ s16 type;
+    /* 0x18 */ Vec3f vel;
```

Comment:
We want to standardize some of these names, so `active` should be `isEnabled`, and `vel` should be `velocity`. The rest of the names match our standards. From what I can tell, Egol and Estone have the same names, so they should also be updated.

Reply:
Is `velocity` standard? There are a lot of actors that use `vel`.

Reply:
`velocity` is the current standard for effects. I think a separate discussion could be had for changing it to `vel`, but for the purposes of this PR, I'd rather stick to what's standard and then potentially change it all later in one big sweep.

Reply:
Yeah, we'll probably change to `vel` eventually, but everything is `velocity` for now.